test/dependency_source/test_poetry.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from packaging.version import Version
+
+from pip_audit._dependency_source import PoetrySource
+from pip_audit._service import ResolvedDependency, SkippedDependency
+
+TEST_DATA_PATH = Path(__file__).parent / "data"
```

Comment:
Another tiny nitpick: could you turn this into an `asset` or `test_data` fixture and put it in `conftest.py`?

Here's the pattern we use on a handful of other projects:

```python
# conftest.py

_ASSETS = (Path(__file__).parent / "assets").resolve()
assert _ASSETS.is_dir()

@pytest.fixture
def asset():
    def _asset(name: str) -> Path:
        return _ASSETS / name

    return _asset
```

Reply:
I could. But why? This one is short, type-safe, and not coupled on conftest.py. I love good autocomplete.

Reply:
My main reason is potential reuse: we have a lot of other tests (particularly the requirements file ones) that would benefit from being broken out into stand-alone assets, so having a centralized "asset" fixture would keep things unified. 

Separately (and I admit I have no idea if this is "good" practice or not): my understanding is that `pytest` conceptually encourages each test module to contain nothing _except_ test code, with `conftest.py` being the idiomatic place to put any scaffolding.

Reply:
> we have a lot of other tests (particularly the requirements file ones) that would benefit from being broken out into stand-alone assets

YAGNI ;)

> Separately (and I admit I have no idea if this is "good" practice or not): my understanding is that pytest conceptually encourages each test module to contain nothing except test code

Never heard of that. Test files can contain fixtures, and from my experience, they should when the fixture is used only in that file. Having all fixtures in `conftest.py` brings unnecessary coupling, makes it harder to track which fixtures aren't needed anymore, and that file being a typical example of "God class" (in this case, God module) can grow to some crazy numbers (last month, I decoupled 10k lines conftest.py). And even if we assume "keep all fixtures in conftest.py", it's not a fixture. It's literally a constant.

Having a constant has direct benefits: it's shorter, type-safe, closer to the code (easier to unravel the test code), and makes the test file self-contained. If none of that persuades you, please, just say it, and I'll update the PR. I want the code I contribute to be nice and readable, but the final word is always on the direct maintainers.